src/core/channel/client_setup.c

```diff
@@ -180,42 +214,43 @@ void grpc_client_setup_create_and_attach(
   s->in_alarm = 0;
   s->in_cb = 0;
   s->cancelled = 0;
+  grpc_pollset_set_init(&s->interested_parties);
 
   grpc_client_channel_set_transport_setup(newly_minted_channel, &s->base);
 }
 
-int grpc_client_setup_request_should_continue(grpc_client_setup_request *r) {
+int grpc_client_setup_request_should_continue(grpc_client_setup_request *r,
+                                              const char *reason) {
```

Comment:
Just checking: is "reason" meant for future use? It not currently referenced anywhere in the function body.